frontend/src/scenes/surveys/SurveyCustomization.tsx

```diff
@@ -50,14 +54,20 @@ export function Customization({
                         value={appearance?.backgroundColor}
                         onChange={(backgroundColor) => onAppearanceChange({ ...appearance, backgroundColor })}
                         disabled={!surveysStylingAvailable}
+                        className={clsx(validationErrors?.backgroundColor ? 'border-danger' : 'ignore-error-border')}
```

Comment:
make a const for `ignore-error-border`? its been used in many places

Reply:
added